<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_right">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>611</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>initialize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_at">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>at</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_capacity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>640</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>capacity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>640</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>680</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>710</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_empty">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>empty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_maxSize">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>max size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_popBack">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>640</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>pop back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_size">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>640</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_data">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>640</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pushBack">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>PushBack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resize">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>640</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reserve">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>reserve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_erase">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>640</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_begin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_end">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>700</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>end</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
